srcpkgs/veikk-tablet/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'veikk-tablet'
+pkgname=veikk-tablet
+version=3.5.8
+_pkgname=vktablet
+revision=3
+_pkgv=${version}-${revision}
```

Comment:
The `revision` variable is voids internal revision, we increase it if we need to rebuild/update when there is no update. The `-3` is part of the upstream version, since we don't allow `-` in versions, you can replace it with some other character like `p` and then replace it later when you need the original version `distfiles="https://veikk.com/image/catalog/Software/vktablet-${version/p/-}.x86_64.zip"